com.playeveryware.eos/CHANGELOG.md

```diff
@@ -4,6 +4,282 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Changelog
 
+## [3.3.2] - 2024-08-27
+
+### Added
+
+- Git Actions
```

Comment:
This isn't following our change log format: we don't have subsections under each header.